lib/index.js

```diff
@@ -296,9 +309,26 @@ exports.parseGeneral = BBPromise.method(function(chtml){
 	// Copy key-value pairs with defined values to meta
 	var meta = {};
 	var value;
+	var notEmpty = false;
 	Object.keys(clutteredMeta).forEach(function(key){
+
 		value = clutteredMeta[key];
-		if (value){ // Only add if has value
+		var innerValue;
+		if (value !== null && typeof value === 'object'){
+			var arrlength = value.length;
+			for (var i = 0; i < arrlength; i++){
+				var definedValue = {};
+				Object.keys(value[i]).forEach(function(objectProperty){
+					innerValue = value[i][objectProperty];
+					if(innerValue !== undefined){
```

Comment:
I think (!innerValue) is better here. It's actually fairly common for innerValue to be '', which would return true, but not be a valid value.

Reply:
innerValue is a string variable so doing (!innerValue) wouldn't work if that's what you meant. Would it be easier to add innerValue !== "" to the if statement?

Reply:
Whoops, meant if(innerValue), not !innerValue :). My point is that if value[i][objectProperty] === '', then we don't actually want the key in the object. We also don't want the key in the object if it's undefined. So just (innerValue) should be good.